arduino/sketch_read_transmit_from_sd.ino

```diff
@@ -0,0 +1,60 @@
+#include <SD.h>  // Load SD library
+#include <SPI.h>
+
+int chipSelect = 4;  // chip select pin for the MicroSD Card Adapter
+File file;  // File object that is used to read and write data
+
+/*
+ * Reads from SD card and transmit it over the serial output channel line by line with '\n' as the line delimitter. Skips null lines.
+ */
+void readFromSDCardToSerialOutputLineByLine(){    // delay value can be passed in as an argument
+  Serial.print("Initializing SD card...");
+  if(!SD.begin(chipSelect)){    // Try initializing the SD card
+    Serial.println("Initialization Failed");   // if return value is false, something went wrong
+    return;
+  }
+  Serial.println("Initialization Successful");
+
+  file = SD.open("sample.csv");   // Grabing the file reference object
+  if(!file){                      // If the file doesn't exist this would return false
+    Serial.println("error opening sample.csv");
+    return;
+  }
+
+  Serial.println("sample.csv");   
+
+  // read from the file until there's nothing else in it
+  String line;
+  // we could use something here like line.reserve(numberOfBytes) to avoid heap memory fragmentation
+  while(file.available()){
+    line = file.readStringUntil('\n');
+    line.trim();
+    if(line != ""){
+      Serial.println(line);
+      // Delay can be added here to transmit data over the serial channel at a constant rate.
+    }  // Skip blank null lines.
+  }
+  file.close();
+}
+
+// the setup routine runs once when you press reset:
+void setup() {
+  // initialize serial communication at 9600 bits per second:
+  Serial.begin(9600);
```

Comment:
@suresh-gandhi Make a default variable for this rate instead of hardcoding. Later we need to take care of getting the value from a spinner in the Test mode.

Reply:
Yeah, we can. Will do it 👍